sdk/containerregistry/azure-containerregistry/README.md

```diff
@@ -56,12 +56,89 @@ For more information please see [Container Registry Concepts](https://docs.micro
 
 ## Examples
 
-<!-- Pending Sample Creation -->
+The following sections provide several code snippets covering some of the most common ACR Service tasks, including:
+
+- [List repositories](#list-repositories)
+- [List tags with anonymous access](#list-tags-with-anonymous-access)
+- [Set artifact properties](#set-artifact-properties)
+- [Delete images](#delete-images)
+
+Please note that each sample assumes there is a `CONTAINERREGISTRY_ENDPOINT` environment variable set to a string containing the `https://` prefix and the name of the login server, for example "https://myregistry.azurecr.io".
+
+### List repositories
+
+Iterate through the collection of repositories in the registry.
+
+```python
+end_point = os.environ["CONTAINERREGISTRY_ENDPOINT"]
+client = ContainerRegistryClient(end_point, DefaultAzureCredential(), audience="https://management.azure.com")
+
+with client:
+    # Iterate through all the repositories
+    for repository_name in client.list_repository_names():
+        print(repository_name)
+
+client.close()
```

Comment:
If you use `with`, you don't need to close. Those are the two prefered syntaxes:

syntax 1
```python
with ContainerRegistryClient(end_point, DefaultAzureCredential()) as client:
   # do something
```

Syntax 2
``` python
client = ContainerRegistryClient(end_point, DefaultAzureCredential())
# do something
client.close()
```

The first one is prefered, since it will close in case of exception, but the second is acceptable